javascript-client/test/api/OrderApi.spec.js

```diff
@@ -49,27 +51,32 @@
       object[property] = value;
   }
 
-  describe('OrderApi', function() {
-    describe('orderAddPartIdPost', function() {
-      it('should call orderAddPartIdPost successfully', function(done) {
-        //uncomment below and update the code to test orderAddPartIdPost
-        //instance.orderAddPartIdPost(function(error) {
-        //  if (error) throw error;
-        //expect().to.be();
-        //});
-        done();
+  describe('OrderApi', function () {
+    describe('End to End', function () {
+      it('Non admin user should add order to cart and list all orders', async function (done) {
+        const userToken = await authUtil.loginUser(constants.nonAdminUser.email, constants.nonAdminUser.password);
+
+        await authUtil.setUserToken(instance, userToken, constants.apiAuthenticationName);
+
+        instance.orderAddPartIdPost(constants.partId, function (error, data, res) {
+          if (error) throw error;
+          instance.orderUserGet(function (error, data, res) {
+            expect(res.status).to.be(200);
```

Comment:
Can you do more validations here? Is it possible to check that the order contains that part that was ordered. 
If needed you can call getOrders method.